build/testBuildImages.sh

```diff
@@ -34,17 +29,26 @@ cd "$TESTS_SRC_DIR/$testProjectName"
 
 artifactsDir="$REPO_DIR/artifacts"
 mkdir -p "$artifactsDir"
-diagnosticFileLocation="$artifactsDir/$testProjectName-log.txt"
 
 # Create a directory to capture any debug logs that MSBuild generates
 msbuildDebugLogsDir="$artifactsDir/msbuildDebugLogs"
 mkdir -p "$msbuildDebugLogsDir"
 export MSBUILDDEBUGPATH="$msbuildDebugLogsDir"
-export MSBUILDDISABLENODEREUSE=1
+export COMPlus_DbgEnableMiniDump="1"
```

Comment:
should we add a comment with what this thing is about?

Reply:
Sure, will do